packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-email/main/go/integrationsteps.md

```diff
@@ -85,27 +65,17 @@ if err != nil {
 if lr.Token() != nil {
  //Login completion code
 }
-
- //Additional factors required -  redirect to factors page
-s.cache.Set("loginResponse", lr, time.Minute*5)
-http.Redirect(w, r, "/login/factors", http.StatusFound)
-return
 ```
 
-### 5: Show an option to choose the email factor
-
-The next step is to build a page that allows the user to choose a factor
-to continue the authentication flow.
+Display a list of all of the authenticators that the user has enrolled and are ready for use. For example:
 
-<div class="half">
+<div class="half border">
 
-![Displays an option to choose verification through the email factor](/img/oie-embedded-sdk/oie-embedded-sdk-go-use-case-email-verify-page.png)
+![A Choose Your Authenticator page with the choices of email and phone, and a Next button](/img/oie-embedded-sdk/wireframes/choose-authenticator-email-phone-form-g2r28.png)
```

Comment:
Use a wireframe that only shows the Email authenticator.

Reply:
Why? We have not configured the Org to include only the email authenticator so this uses the same "Choose your authenticator form as all the other variants of this guide.

Reply:
The reason why I said this was because the configuration section instructs them to add the email authenticator and no other authenticator. But then we show them a list of other authenticators which is misleading.